src/app/clients/client-stepper/client-general-step/client-general-step.component.ts

```diff
@@ -76,6 +76,7 @@ export class ClientGeneralStepComponent implements OnInit {
       'externalId': [''],
       'genderId': [''],
       'mobileNo': [''],
+      'emailAddress': ['', Validators.email],
```

Comment:
Since the email address should be a plain text, we shouldn't introduce validations to it. Am I misunderstanding the purpose of this?

Reply:
Yes, you're right but I think that is better to set a basic email validation